trie/trieStorageManager.go

```diff
@@ -663,23 +663,7 @@ func (tsm *trieStorageManager) ShouldTakeSnapshot() bool {
 		return false
 	}
 
-	return isActiveDB(stsm)
-}
-
-func isActiveDB(stsm *snapshotTrieStorageManager) bool {
```

Comment:
where is this logic moved?

Reply:
This was removed. The reason was that when the node starts, it opens as many dbs as necessary. At epoch change the node also manages the opening/closing of epoch storers. So these checks are redundant. As an added bonus, even if something happens and a certain snapshot can not be completed fully from the node's own storage, the data will be synced from other nodes (this will not happen if these checks were still in place because the snapshot would not start at all).

Reply:
great! :+1: